### DISCUSSION TITLE
Poco X3 Pro (Vayu) PMIC

### DISCUSSION THREAD

**Discussion by 9M2PJU** [VR-25/acc#198] [Q&A] [answered]
Hello, 

I need to know, if I install acc for my rooted vayu will it have a chance to kill or break my PMIC ?

i read this 

![image](https://github.com/VR-25/acc/assets/991353/3663e00e-55f6-4cda-8a08-f3ba93fc2a39)

**Comment by VR-25** [comment]
By default, acc shuts down the system if battery level drops to 5%.
To mitigate the issue even further, you may want to use charging voltage control over regular charging switches. For that, run acc -t to see which switches work and acc -ss to select a working voltage switch (if available).

**Comment by 9M2PJU** [comment]
Ppl on telegram said that using ACC charging controller will trigger the PMIC bug on Poco X3 Pro.

**Comment by VR-25** [comment, marked as answer]
https://github.com/VR-25/acc#warnings
